test/e2e/tls_test.go

```diff
@@ -131,27 +134,30 @@ func (data *TestData) opensslConnect(t *testing.T, pod string, container string,
 	}{
 		{
 			clusterInfo.podV4NetworkCIDR != "",
-			"127.0.0.1",
+			nodeIPv4,
 			"-4",
 		},
 		{
 			clusterInfo.podV6NetworkCIDR != "",
-			"::1",
+			nodeIPv6,
 			"-6",
 		},
 	}
 	for _, c := range opensslConnectCommands {
 		if !c.enabled {
 			continue
 		}
-		cmd := []string{"timeout", "1", "openssl", "s_client", "-connect", net.JoinHostPort(c.ip, fmt.Sprint(port)), c.option}
+		cmd := []string{"curl", "-k", "-v", fmt.Sprintf("https://%s", net.JoinHostPort(c.ip, fmt.Sprint(port)))}
 		if tls12 {
-			cmd = append(cmd, "-tls1_2")
+			cmd = append(cmd, "--tls-max", "1.2", "--tlsv1.2")
 		}
 		stdout, stderr, err := data.RunCommandFromPod(antreaNamespace, pod, container, cmd)
-		assert.NoError(t, err, "failed to run openssl command on Pod '%s'\nstderr: %s", pod, stderr)
+		assert.NoError(t, err, "failed to run openssl command on Pod '%s'\nstderr: %s", pod, stdout)
```

Comment:
```suggestion
		assert.NoError(t, err, "failed to run curl command on Pod '%s'\nstdout: %s", pod, stdout)
```
No need to check the stderr?

Reply:
we are already checking stderr in checkTLS function.